coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The `coverage` directory should not be present in the repository, therefore must be added to the `.gitignore` and recursively removed from the repo.